imagersite/imagersite/tests.py

```diff
@@ -0,0 +1,148 @@
+import unittest
+from django.test import Client
+from imager_images.models import Photo
+from imager_profile.models import User
+import factory
+from faker import Factory as FakeFaker
+from django.conf import settings
+import os
+from django.core.mail import outbox
+import re
+
+fake = FakeFaker.create()
+
+
+class PhotoFactory(factory.Factory):
+    class Meta:
+        model = Photo
+    title = fake.sentence()
+    description = fake.paragraph()
+
+
+class UserFactory(factory.Factory):
+    class Meta:
+        model = User
+    first_name = fake.first_name()
+    last_name = fake.last_name()
+    email = factory.LazyAttribute(lambda a: '{}.{}@example.com'.format(
+                                  a.first_name, a.last_name).lower())
+    username = factory.LazyAttribute(lambda a: '{}{}{}{}'.format(
+                                     a.last_name[:1], a.first_name[1:],
+                                     a.first_name[:1], a.last_name[1:]).lower())
+
+
+class HomeExists(unittest.TestCase):
+    """Test for 200 OK at '/'"""
+    def setUp(self):
+        self.client = Client()
+
+    def test_self_exists(self):
+        response = self.client.get('/')
+        self.assertEqual(response.status_code, 200)
+
+    def test_links_for_home(self):
+        response = self.client.get('/')
+        self.assertIn('<a href="/login/">login</a>', response.content)
+        self.assertIn('<a href="/register/">register</a>', response.content)
+
+
+class HomeRandomPhotos(unittest.TestCase):
+    """Test for random photo selection on website"""
+    def setUp(self):
+        self.client = Client()
+        owner = UserFactory.create()
+        owner.save()
+        self.photo1 = PhotoFactory.create(owner=owner, privacy='PU',
+            file=os.path.join(settings.MEDIA_ROOT, 'amoosing.jpg'))
+        self.photo2 = PhotoFactory.create(owner=owner, privacy='PU',
+            file=os.path.join(settings.MEDIA_ROOT, 'googlephoto.jpg'))
+        self.photo1.save()
+        self.photo2.save()
+
+    def tearDown(self):
+        self.photo1.delete()
+        self.photo2.delete()
```

Comment:
Ok, thanks! I was surprised that I wasn't getting database isolation, but I didn't realize there was a Django class of the same name.